### PR TITLE
Ra dspdc 795 table ingest template

### PR DESCRIPTION
**Pull request by raaidbroad**
Successful run here: https://argo.monster-dev.broadinstitute.org/workflows/clinvar/clinvar-ingest-826ss?tab=workflow

added table ingest piece, fixed up the poll-ingest-job script too.

### CODE REVIEW

File: orchestration/scripts/submit-table-ingest.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        raise HTTPError(f"Bad response, got code of: {response.status_code}")
+
+# print the job id to std out
+print(ingest_table(dataset_id,
+                   format="json",
+                   ignore_unknown_values=False,
+                   max_bad_records=0,
+                   path=f"{source_path}/*",
+                   table=table_name
+                   ))
```
Comment (danxmoran) [inline orchestration/scripts/submit-table-ingest.py:30]: Nit: Wot formatting is dis?

File: orchestration/scripts/submit-table-ingest.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+base_url = os.environ["API_URL"]
+dataset_id = os.environ["DATASET_ID"]
+source_path = os.environ["SOURCE_PATH"]
+table_name = os.environ["TABLE_NAME"]
+
+authed_session = AuthorizedSession(credentials)
+
+
+def ingest_table(dataset_id: str, **kwargs):
+    response = authed_session.post(f"{base_url}/api/repository/v1/datasets/{dataset_id}/ingest", json=kwargs)
```
Comment (danxmoran) [inline orchestration/scripts/submit-table-ingest.py:17]: Majick!

File: orchestration/templates/ingest-table.yaml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+          - name: source-path
+          - name: table-name
+          - name: sa-secret
+          - name: sa-secret-key
+      volumes:
+        - name: '{{ $secretVolume }}'
+          secret:
+            secretName: '{{ .Values.repo.accessKey.secretName }}'
+      script:
+        image: us.gcr.io/broad-dsp-gcr-public/monster-auth-req-py:1.0.0
```
Comment (danxmoran) [inline orchestration/templates/ingest-table.yaml:70]: Should these be `1.0.0`? I thought we had a newer version

File: orchestration/scripts/submit-table-ingest.py (hunk 1 above)
Comment (raaidbroad) [reply orchestration/scripts/submit-table-ingest.py:30]: idk lol, I'll copy it to pycharm to format real quick

File: orchestration/scripts/submit-table-ingest.py (hunk 2 above)
Comment (raaidbroad) [reply orchestration/scripts/submit-table-ingest.py:17]: we love kwargs

File: orchestration/templates/ingest-table.yaml (hunk 3 above)
Comment (raaidbroad) [reply orchestration/templates/ingest-table.yaml:70]: YES Good catch

File: orchestration/scripts/submit-table-ingest.py (hunk 1 above)
Comment (raaidbroad) [reply orchestration/scripts/submit-table-ingest.py:30]: pycharm says it is already properly formatted??

Comment (danxmoran) [reply orchestration/scripts/submit-table-ingest.py:30]: Would pull the dangling parens up, but nbd

**Pull request merged**